File: src/main/java/me/miki/shindo/management/event/impl/EventRenderScoreboard.java
package me.miki.shindo.management.event.impl;

import me.miki.shindo.management.event.Event;
import net.minecraft.scoreboard.ScoreObjective;

public class EventRenderScoreboard extends Event {

    private final ScoreObjective objective;

    public EventRenderScoreboard(ScoreObjective objective) {
        this.objective = objective;
    }

    public ScoreObjective getObjective() {
        return objective;
    }
}

File: src/main/kotlin/me/miki/shindo/management/event/impl/EventRenderScoreboard.kt
package me.miki.shindo.management.event.impl

import me.miki.shindo.management.event.Event
import net.minecraft.scoreboard.ScoreObjective

class EventRenderScoreboard(
    private val _objective: ScoreObjective,
) : Event() {
    fun getObjective(): ScoreObjective = _objective
}
